tools/cli/services/wl_template_manager.py

```
@@ -150,12 +161,14 @@ def update(self):
         if not self.wl_repo:
             self.wl_repo = Repo(self.wl_repo_folder)
         with self.wl_repo.git.custom_environment(
-                GIT_SSH_COMMAND=f"ssh -o StrictHostKeyChecking=no -i {self.ssh_pkey_path}"):
+            GIT_SSH_COMMAND=f"ssh -o StrictHostKeyChecking=no -i {self.ssh_pkey_path}"):
```

Flake8 (7.2.0)

[error] continuation line with same indent as next logical line

(E125)